Lib/concurrent/futures/process.py

```diff
@@ -439,6 +448,7 @@ def shutdown_worker():
                 # this thread if there are no pending work items.
                 if not pending_work_items:
                     shutdown_worker()
+                    thread_wakeup = None
```

Comment:
Is this required?

Reply:
As we return just after, it is not necessary indeed.